docker-stack.yml

```
@@ -78,8 +78,8 @@ services:
                 django_tables_exist=$$?
 
                 if [ $$django_tables_exist -gt 0 ] || [ $$REFRESH_DB == 'true' ]; then
-                    ./cfgov/manage.py update_index
-                    ./refresh-data.sh && httpd -d cfgov/apache -D FOREGROUND
+                    ./refresh-data.sh 
+                    ./cfgov/manage.py update_index && httpd -d cfgov/apache -D FOREGROUND
```

Since we have multiple lines here now, let's just put each command on its own line.

```suggestion
                    ./cfgov/manage.py update_index
                    httpd -d cfgov/apache -D FOREGROUND
```